README.md

```diff
@@ -2,186 +2,139 @@
-      * [Manually refreshing access token](#manually-refreshing-access-token)
-    * [Using API keys](#using-api-keys)
-    * [Using JWT (Service Tokens)](#using-jwt-service-tokens)
-    * [Choosing the correct credential type automatically](#choosing-the-correct-credential-type-automatically)
   * [Specifying request body](#specifying-request-body)
   * [Media uploads](#media-uploads)
-    * [Example: Upload a plain text file to Google Drive](#example-upload-a-plain-text-file-to-google-drive)
-    * [Example: Upload an image to Google Drive from a readable stream](#example-upload-an-image-to-google-drive-from-a-readable-stream)
-  * [Exposing request object](#exposing-request-object)
-  * [Options](#options)
-    * [Available options](#available-options)
-    * [Global options](#global-options)
-      * [Example: Specifying a default `timeout` and `auth` to be used for each request](#example-specifying-a-default-timeout-and-auth-to-be-used-for-each-request)
-      * [Example: Specifying global request parameters](#example-specifying-global-request-parameters)
-    * [Service-client options](#service-client-options)
-      * [Example: Specifying a default `auth` option (API key or OAuth2 client)](#example-specifying-a-default-auth-option-api-key-or-oauth2-client)
-      * [Example: Specifying default service client query parameters](#example-specifying-default-service-client-query-parameters)
-    * [Request-level options](#request-level-options)
-* [Using a Proxy](#using-a-proxy)
+  * [Request Options](#request-options)
+  * [Using a Proxy](#using-a-proxy)
 * [License](#license)
 * [Contributing](#contributing)
+* [Questions/problems?](#questionsproblems)
 
-### Alpha
-
-This library is in Alpha. We will make an effort to support the library, but we reserve the right to make incompatible changes when necessary.
-
-### Release Notes & Breaking Changes
-
-You can find a detailed list of breaking changes and new features in our [Release Notes][releasenotes]. If you've used this library before `25.x`, see our [Release Notes][releasenotes] to learn about migrating your code from `24.x.x` to `25.x.x`. It's pretty easy :)
-
-### Supported APIs
-
+## Google APIs
 The full list of supported APIs can be found [here][supported-list]. The API endpoints are automatically generated, so if the API is not in the list, it is currently not supported by this API client library.
 
-### Questions/problems?
-
-* Ask your development related questions on [Ask a question on Stackoverflow][stackoverflow]
-* If you've found an bug/issue, please [file it on GitHub][bugs].
+Supported APIs are listed on the [Google APIs Explorer][apiexplorer].
 
 ### Working with Google Cloud Platform APIs?
-
 If you're working with [Google Cloud Platform][cloudplatform] APIs such as Datastore, Cloud Storage or Pub/Sub, consider using the [`google-cloud`][googlecloud] package, an idiomatic Node.js client for Google Cloud Platform services.
```

Comment:
There should be no mention of `google-cloud` package as it's now decoupled into many small packages and the original monorepo is not updated. [This](https://cloud.google.com/nodejs/docs/reference/libraries) is the list of `@google-cloud` packages that are documented.